index.html

```diff
@@ -110,6 +110,28 @@ <h3 class="works_headline">My Recent Works</h3>
           class="works_headline_indicator"
         />
       </div>
+      <!-- CARD WORK PROJECT DETAILS -->
+      <div id="work_project_details" class="work_project_details">
+        
+        <div class="work_project_details_card">
+          <span class="work_detail_close_icon">&times; </span>
+          <img class="work_project_details_img" src="./media/images/projects/youtube.PNG" alt="project screenshot"/>
+            <div class="work_project_details_title_btns"><h5 class="work_project_details_title"></h5>
+              <div class="work_project_details_btn_container btns_title">
+                <button type="button" class="buttons btns_title_live">See Live <img src="./media/Icons/seeliveicon.svg" alt="see live icon"/></button>
+                <button type="button" class="buttons btns_title_github">See Source <img src="./media/Icons/Icon-GitHub.svg" alt="see github icon"/></button>
+              </div>
+            </div>
```

Comment:
- [ ] In figma you are provided with specifications on text design, you can find them on right side menu:

![image](https://user-images.githubusercontent.com/77038610/151188624-c4e77bb5-0bd5-4b3c-b244-510980aa9bea.png)

Please make sure to match the design of paragraph by giving the same specifications.